### PR TITLE
Update ZoomControls.qml

### LINKED ISSUE #350: webview.zoomFactor is not always changed on first time
There is QT Bug: https://bugreports.qt.io/browse/QTBUG-84313
During my tests on the "xenial_-_nohighdpiautoscale" branch, I've seen that zoom in / zoom out actions (Zoom menu) are sometimes just ignored, the webview.zoomFactor value was unchanged.

The workaround is to set the zoom factor twice.

### LINKED ISSUE #360: Video too big / too small in fullscreen mode, when zoomFactor != 1.0
- go to a page that has a video in landscape format that can be put to full screen
- device is in landscape orientation

1)
- set the zoom to 50% 
- set video to full screen
expected result: the video covers the whole screen
actual result: video is smaller (covers only part of the screen)

2)
- set the zoom to 150% 
- set video to full screen
expected result: the video covers the whole screen
actual result: video is bigger (only part of the video is displayed on the screen)

### PR DESCRIPTION
**Pull request by balcy**
1)
workaround for zoom changes (https://bugreports.qt.io/browse/QTBUG-84313)
fixes https://github.com/ubports/morph-browser/issues/350

2)
fixes https://github.com/ubports/morph-browser/issues/360

### CODE REVIEW

File: src/app/WebViewImpl.qml [hunk 1]
```diff
@@ -699,6 +699,14 @@ WebView {
         }
 
     onFullScreenRequested: function(request) {
+        if (webview.isFullSceeen) {
+          webview.zoomController.refresh();
+        }
+        else {
+          // twice because of QTBUG-84313
+          webview.zoomFactor = 1.0;
+          webview.zoomFactor = 1.0;
```
Comment (UniversalSuperBox) [inline src/app/WebViewImpl.qml:708]: Why does zoom need to be reset to 1 when leaving fullscreen? Is this bypassing zoomController?

File: src/app/webbrowser/Browser.qml [hunk 2]
```diff
@@ -844,16 +844,16 @@ Common.BrowserView {
             target: browser.currentWebview
             onLoadingChanged: {
                 if (browser.currentWebview.loading) {
-                    chrome.state = "shown"
+                    chrome.state = "shown";
                 } else if (browser.currentWebview.isFullScreen) {
-                    chrome.state = "hidden"
+                    chrome.state = "hidden";
```
Comment (UniversalSuperBox) [inline src/app/webbrowser/Browser.qml]: These changes, while useful, are not necessary for this PR. It'd be best if they could be removed from the commit they are added, or split out into their own commit so they're at least atomic.

**Review by UniversalSuperBox — CHANGES_REQUESTED** [review]

File: src/app/WebViewImpl.qml (hunk 1 above)
Comment (balcy) [reply src/app/WebViewImpl.qml:708]: actually it is set to 1 when entering fullscreen. (see https://github.com/ubports/morph-browser/issues/360)
If the current state is fullscreen, it is about to leave fullscreen mode.

Maybe the following would improve the readability ? 
```
onFullScreenRequested: function(request) {
        if (request.toggleOn) {
            // twice because of QTBUG-84313
            webview.zoomFactor = 1.0;
            webview.zoomFactor = 1.0;
        }
        else {
            webview.zoomController.refresh();
        }
        request.accept();
    }
```

Comment (balcy) [reply src/app/WebViewImpl.qml:708]: the zoom factor of the zoom controller remains unchanged, only in fullscreen mode the browser zoom goes to 1.0
So it is bypassed while in fullscreen mode

File: src/app/webbrowser/Browser.qml (hunk 2 above)
Comment (balcy) [reply src/app/webbrowser/Browser.qml]: luckily it was like the example in https://stackoverflow.com/a/9103397 : )
So I've removed the changes from the loadingChanged function.

**Review by UniversalSuperBox — APPROVED** [review]
Looks good and resolves the listed issues.
